### PR TITLE
Choim nse5718

### PR DESCRIPTION
**Pull request by choim-sci**
@covesturtevant, 

This is PR to review the unit test for wrap.loc.asgn.R.   Thanks!

### CODE REVIEW

File: flow/flow.loc.asgn/tests/testthat/test-wrap-loc-asgn.R [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    unlink(testOutputDir, recursive = TRUE)
+  }
+  returnedOutputDir <- wrap.loc.asgn(
+    DirIn = testInputDir,
+    DirOutBase = testOutputDir,
+    TimeBgn = as.POSIXct('2019-01-01', tz = 'GMT'),
+    TimeEnd = as.POSIXct('2019-01-06', tz = 'GMT'),
+    TypeFile = 'namedLocation'
+  )
+  testthat::expect_true (dir.exists(testOutputDirPath))
```
Comment (covesturtevant) [inline flow/flow.loc.asgn/tests/testthat/test-wrap-loc-asgn.R:118]: This test fails because the location file does not exist. But I think this test is a duplicate of Test Scenario 1. Delete?

**Review by covesturtevant — CHANGES_REQUESTED** [review]
@choim-sci Tests generally look great. A couple requests for revision:

- Would you please check out the comment I made in the test script (line 119) about the missing test file and/or duplicate test? 
- It's great that the tests check for the output directory to be present, but I think they should go one step further and check that that output file is present. Could you add this? 

Thanks!

File: flow/flow.loc.asgn/tests/testthat/test-wrap-loc-asgn.R (hunk 1 above)
Comment (choim-sci) [reply flow/flow.loc.asgn/tests/testthat/test-wrap-loc-asgn.R]: 1. Oh, I forgot to add the file, /locations/prt/24688/prt_24688_locations.json.  But doesn't look like it is needed for the test. 

2. For the test 3, I modified it to be a negative; in case TypeFile = 'namedLocation' but the input json has missing "active_dates" Let me know if this is not necessary.  (this error is not caught atm.)

3.  checking the presence of the output file is added

**Comment by covesturtevant** [comment]
Looks great @choim-sci! Thanks for the fixes.
